pkgs/os-specific/linux/util-linux/default.nix

```diff
@@ -72,4 +76,6 @@ stdenv.mkDerivation rec {
     platforms = platforms.linux;
     priority = 6; # lower priority than coreutils ("kill") and shadow ("login" etc.) packages
   };
-}
+} // lib.optionalAttrs stdenv.hostPlatform.isStatic {
+  NIX_CFLAGS_LINK = "-laudit";
```

Comment:
Since this is too big for master anyway, why not move it inside?

Reply:
Should I rebase to staging ?

Reply:
Fixed, should I rebase to staging ?

Reply:
> Fixed, should I rebase to staging ?

Yes please.

Reply:
Fixed.